server.js

```diff
@@ -172,10 +182,65 @@ app.post('/getParticularShoutout', (req, res, next) => {
   // }
   db.loadParticular('shoutout', req.body.shoutoutInfo)
     .then((data) => {
-      res.end('Here is the requested shoutout: \n' + JSON.stringify(data, undefined, 2));
+      res.end(JSON.stringify(data));
     })
 });
 
+
+
+// ===============================
+// ===== Generic Update Path =====
+// ===============================
+
+// Unlike the specific paths, this path accepts the model type as a part of the request body
+// This way the server code doesn't have to have unnecessarily duplicated data
+app.patch('/updateComments', (req, res, next) => {
+  // Example postman requests:
+  // -------------------------
+
+  // === Student ===
+  /*
+  {
+	"modelType" : "student",
+	"identifier": {
+		"_id": "5a0cfe51085065bed328591b"
+	},
+	"comment": {
+		"name": "Vi",
+		"comment": "I know like 50 languages"
+	}
+}
+  */
+
+  // === Event ===
+  /*
+  {
+    "modelType" : "event",
+    "identifier": {
+      "_id": "5a0dd8307bd6aac9c8394636"
+    },
+    "comment": {
+      "name": "Phil",
+      "comment": "I can't do sign language correctly because I have crazy old man arthritis"
```

Comment:
Is the arthritis 'crazy' or the old man?

Reply:
While it may be a dangling participle, it does work pretty well for both, so I'm good with leaving it.

Reply:
Asking the real questions